tigerpath/majors_and_certificates/majors/MAT_2018.json

```diff
@@ -106,7 +106,7 @@
           ]
         },
         {
-          "name": "Geometry or Topology",
+          "name": "Geometry/ Topology or Discrete Math",

```

Comment:
Maybe
```suggestion
          "name": "Geometry, Topology, or Discrete Math",
```

Reply:
I tried testing it and I think if you put a space it doesn't break it, but yeah maybe we could stick to the commas just in case. But Geometry and Topology is apparently very similar according to feedback and it makes more sense to be Geometry/Topology or Discrete Math. It's hard to do without using special characters

Reply:
I think it's fine to have commas for now. No one will complain.
On the other hand, I could definitely see '/' causing real problems, and it looks bad that it's not symmetric.

On the longer term, of course, we need to escape our API calls and req paths in particular.